doc/source/user/tutorial-svd.rst

```diff
@@ -0,0 +1,507 @@
+
+**Learner profile**
+
+This tutorial is for people who have a basic understanding of linear
+algebra and arrays in NumPy and want to understand how n-dimensional
+(:math:`n>=2`) arrays are represented and can be manipulated. In particular, if
+you don't know how to apply common functions to n-dimensional arrays (without
+using for-loops), or if you want to understand axis and shape properties for
+n-dimensional arrays, this tutorial might be of help.
+
+**Learning Objectives**
+
+After this tutorial, you should be able to:
+
+- Understand the difference between one-, two- and n-dimensional arrays in
+  NumPy;
+- Understand how to apply some linear algebra operations to n-dimensional
+  arrays without using for-loops;
+- Understand axis and shape properties for n-dimensional arrays.
+
+**Content**
+
+In this tutorial, we will use a `matrix decomposition
+<https://en.wikipedia.org/wiki/Matrix_decomposition>`_ from linear algebra, the
+Singular Value Decomposition, to generate a compressed approximation of an
+image. We'll use the ``face`` image from the `scipy.misc` module:
+
+    >>> from scipy import misc
+    >>> img = misc.face()
+
+.. note::
+
+   If you prefer, you can use your own image as you work through this tutorial.
+   In order to transform your image into a NumPy array that can be manipulated,
+   you can use the ``imread`` function from the `matplotlib.pyplot` submodule.
+   Alternatively, you can use the :func:`imageio.imread` function from the
+   ``imageio`` library. Be aware that if you use your own image, you'll likely
+   need to adapt the steps below. For more information on how images are treated
+   when converted to NumPy arrays, see :std:doc:`user_guide/numpy_images` from
+   the ``scikit-image`` documentation.
+
+Now, ``img`` is a NumPy array, as we can see when using the ``type`` function::
+
+    >>> type(img)
+    numpy.ndarray
+
+We can see the image using the `matplotlib.pyplot.imshow` function::
+
+    >>> import matplotlib.pyplot as plt
+    >>> plt.imshow(img)
+
+.. plot:: user/plot_face.py
+    :align: center
+    :include-source: 0
+
+.. note::
+
+   If you are executing the commands above in the IPython shell, it might be
+   necessary to use the command ``plt.show()`` to show the image window. 
+		     
+**Shape, axis and array properties**
+
+Note that, in linear algebra, the dimension of a vector refers to the number of
+entries in an array. In NumPy, it instead defines the number of axes. For
+example, a 1D array is a vector such as ``[1, 2, 3]``, a 2D array is a matrix,
+and so forth.
+
+First, let's check for the shape of the data in our array. Since this image is
+two-dimensional (the pixels in the image form a rectangle), we might expect a
+two-dimensional array to represent it (a matrix). However, using the ``shape``
+property of this NumPy array gives us a different result::
+
+    >>> img.shape
+    (768, 1024, 3)
+
+The output is a `tuple` with three elements, which means that this is a
```

Comment:
`tuple` does not render as a link. You probably want 
```
:ref:`tuple <tut-tuples>`
```
which goes to [here](https://docs.python.org/dev/tutorial/datastructures.html#tut-tuples)

Reply:
`python:tut-tuples` would be clearer.

Reply:
I was actually thinking of emphasis for those, but I'll add the links to make it clearer. Thanks!